src/map.ts

```diff
@@ -1,9 +1,8 @@
+import { isCached } from './index';
 import { mapLoadable } from './bundler';
-import { RequireLoader, ImportLoader } from './interface';
+import { RequireLoader, ImportLoader, CachedComponent } from './interface';
 
-const cache = {} as any;
-
-export const isCached = (componentName: string) => !!cache[componentName];
+export const cache = new Map<string, CachedComponent>();
```

Comment:
Overall - why do we need to export `cache`?

I think the purpose of this file was to incapsulate a work with internal `Map` interface and provide only methods to work with cache, i. e. `getComponent`/`getComponentFromCache` (in future, like now, you just need to change internal methods and all other code shouldn't be changed - that was the purpose of all these methods).